api/src/main/scala/hmda/api/http/LarHttpApi.scala

```diff
@@ -46,21 +48,23 @@ trait LarHttpApi extends LarProtocol with ValidationResultProtocol with HmdaCust
   val validateLarRoute =
     pathPrefix("lar") {
       path("validate") {
-        parameters('check.as[String] ? "all") { (checkType) =>
-          timedPost {
-            entity(as[LoanApplicationRegister]) { lar =>
-              val larValidation = system.actorSelection("/user/larValidation")
-              val checkMessage = checkType match {
-                case "syntactical" => CheckSyntactical(lar, ValidationContext(None))
-                case "validity" => CheckValidity(lar, ValidationContext(None))
-                case "quality" => CheckQuality(lar, ValidationContext(None))
-                case _ => CheckAll(lar, ValidationContext(None))
-              }
-              onComplete((larValidation ? checkMessage).mapTo[List[ValidationError]]) {
-                case Success(xs) =>
-                  complete(ToResponseMarshallable(xs))
-                case Failure(e) =>
-                  complete(HttpResponse(StatusCodes.InternalServerError))
+        hmdaAuthorize {
```

Comment:
Do we want to _require_ authorization for `/lar/validate`?  It seems like it should be optional.  In addition, any institution data in the header is not being passed on to the underlying validation engines.  To do so, you'd need to add `ValidationContext(institution)` instead of `ValidationContext(None)`

Also, the current header allows for multiple institutions, while these require a specific Institution to be set.  Seems like we should leave this unauthorized until we figure out how best to deal with this.